src/03_modules.py

```diff
@@ -10,22 +10,29 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+#print(sys.argv)
+for arg in sys.argv:
+    print(arg)
 
 # Print out the OS platform you're using:
 # YOUR CODE HERE
+print(sys.platform)
 
 # Print out the version of Python you're using:
 # YOUR CODE HERE
-
+print(sys.version)
 
 import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
 # YOUR CODE HERE
+print(os.getgid())
```

Comment:
great job; this might actually what you want: `print(os.getpid())`

That command might return something a little different, the current group id.